libp2p-websocket/src/browser.rs

```diff
@@ -206,41 +206,46 @@ impl Transport for BrowserWsConfig {
 	}
 
     fn nat_traversal(&self, server: &Multiaddr, observed: &Multiaddr) -> Option<Multiaddr> {
-        let server_protocols: Vec<_> = server.iter().collect();
-        let observed_protocols: Vec<_> = observed.iter().collect();
+		let mut server_protocols = server.iter();
```

Comment:
looks like this file has inconsistent formatting, line `208` is formatted with spaces, and this one with tabs